tests/llm-as-a-judge/correctness/test_openai_eval.py

```
@@ -0,0 +1,273 @@
+import json
+import os
+import re
+import string
+import subprocess
+from datetime import datetime
+from pathlib import Path
+from typing import Dict, List, Tuple, Any, Optional
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)